### PR TITLE
workflow: update maintainers

### PR DESCRIPTION
**Pull request by 0xc0170**
@NirSonnenschein To complete the update, need your username on mbed.com please

### CODE REVIEW

**Comment by cmonr** [comment]
Oh, if this is being updated, my public profile should probably be replaced with my ARM profile: https://os.mbed.com/users/CM2/

**Comment by 0xc0170** [comment]
Updated

@AnotherButler Ready for review

**Comment by 0xc0170** [comment]
@ARMmbed/mbed-os-maintainers please review

**Comment by AnotherButler** [comment]
Do you mind if I put these in alphabetical order, or is there a reason they're in this order?

File: docs/reference/contributing/guidelines/workflow.md [hunk 1]
```diff
@@ -20,10 +20,9 @@ The current maintainers are:
 
 - [Anna Bridge](https://os.mbed.com/users/AnnaBridge).
 - [Martin Kojtal](https://os.mbed.com/users/Kojto).
-- [Jimmy Brisson](https://os.mbed.com/users/theotherjimmy).
 - [Shrikant Tudavekar](https://os.mbed.com/users/shrikant1213).
-- [Sam Grove](https://os.mbed.com/users/sam_grove).
-- [Cruz Monrreal](https://os.mbed.com/users/MrCruz).
+- [Nir Sonnenshein](https://os.mbed.com/users/nirs).
```
Comment (NirSonnenschein) [inline docs/reference/contributing/guidelines/workflow.md]: Sonnenshein -> Sonnenschein

Comment (0xc0170) [reply docs/reference/contributing/guidelines/workflow.md]: Fixed

Comment (AnotherButler) [reply docs/reference/contributing/guidelines/workflow.md]: @NirSonnenschein Sorry for missing this

**Pull request merged**